### DISCUSSION TITLE
R 5.0.1 Sunbird coKreat - Circle QA Sign Off and Regression cycle sign off for Cokreat BB

### DISCUSSION THREAD

**Discussion by Nadiya496** [Sunbird-coKreat/Community#25] [General Discussions]
Hello All,

This is  Consolidated circle  QA Sign off for the release 5.0.1.0 for Sunbird coKreat Building block. 

Environment: Dock Dev
Testing Status: Completed
Defects Dashboard: NA

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=240901689)

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=12627)

Circle Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1jail-mLcIooaYbRIIRWVeFmXfw90oNoY_g_SYayp2Wg/edit#gid=1985233391)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav,@luckynvdu @rajeevsathish @surendrasinghs @sharathkashyap

**Comment by anunagaraj** [comment]
@Nadiya496  CoKreate Microsite is not updated with the Release notes for R5.0.1. Kindly get the Microsite updated.

CC: @pallakartheekreddy  @ShailandraOjha @chitranshu-keshav @surendrasinghs

**Reply by Nadiya496** [reply]
@anunagaraj It is updated now. Kindly check. https://cokreat.sunbird.org/use/release-notes/cokreat-release-v5.0.1-upcoming

**Reply by Nadiya496** [reply]
Hello All,

This is Consolidated regression cycle QA Sign off for the release 5.0.1.0 for Sunbird coKreat Building block.

Environment: Dock Staging Cokreat portal.
Testing Status: Completed
Defects Dashboard: NA
**Note:** Since there was no QA required tickets, Cokreat sanity is beed done on cokreat portal.

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=240901689)

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=12627)

Circle Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1jail-mLcIooaYbRIIRWVeFmXfw90oNoY_g_SYayp2Wg/edit#gid=1985233391)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav,@luckynvdu @rajeevsathish @surendrasinghs @sharathkashyap

> circle
